### ISSUE TITLE
config file with `upgrade: false` shows no output

### ISSUE THREAD

**Issue by uloco** [raineorshine/npm-check-updates#1100]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 12`

---------------------------

## Steps to Reproduce

Create a `.ncurc.json` with only this
```
{
  "upgrade": false
}

```
## Current Behaviour
Output shows 
```No dependencies.```

## Expected Behaviour
Same behaviour as running `ncu` without any arguments. It should show all the dependencies with their versions etc.

# Additional bug properly related

If I add a filter config to the .ncurc.json file, than I get the following error, although I am just running `npm-check-updates` without any arguments at all.

```
Error: Cannot specify a filter using both --filter and args. Did you forget to quote an argument?
```

**Comment by uloco** [comment]
Removing the upgrade property from the config file completely does work as expected though!

**Comment by raineorshine** [comment]
Thanks for reporting! The cause is that config values are piped back into commander to be parsed, but commander is interpreting `--upgrade false` as two separate cli arguments. I'm working on a fix.

**Comment by raineorshine** [comment]
Fixed and published in `v12.5.10`
